.yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
```

Comment:
I should have maybe pointed this out of the `psammead` PR but are this and `.yarn/releases/yarn-3.0.0.cjs` meant to be source controlled or ignored as per the `git ignore`?

Reply:
In the migration [instructions](https://yarnpkg.com/getting-started/migration#step-by-step) on step 6 it says you should commit the JS and step 12 says you should commit the plugin files, so yes I think.